packages/strapi-admin/admin/src/containers/NotFoundPage/index.js

```diff
@@ -28,8 +29,24 @@ export default class NotFound extends React.Component { // eslint-disable-line r
         <h2 className={styles.notFoundDescription}>
           <FormattedMessage {...messages.description} />
         </h2>
-        <Link to="/admin">Back to home page.</Link>
+        <Button
+          label="Back to home page"
```

Comment:
The label should be translated (pass the translation id as label and add `handlei18n` prop).